src/Microsoft.Azure.WebJobs.Host/app.config

```diff
@@ -1,23 +1,23 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Just in case - redirect Application Insights of earlier versions to the latest to avoid conflicts with the customer code